libs/pageserver_api/src/models.rs

```diff
@@ -242,29 +243,29 @@ pub enum PagestreamBeMessage {
     DbSize(PagestreamDbSizeResponse),
 }
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
I think, clippy used to complain that no `Eq` is derived, but maybe it will do that in newer rustc versions.